MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.google.moviestvsentiments;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
```

Comment:
nit: This is fairly obvious and generally do not need comment. (https://drive.google.com/file/d/1CxWFDHDR0VBk38EgySAdYDDVF2nQ73Nq/view)